scraper/runspiders.sh

```diff
@@ -0,0 +1,11 @@
+cd newsscrapper/spiders
+
+scrapy crawl cnn
+
+for i in {10..1};
+do
+    echo "NEXT SPIDER WILL RUN IN: $i"
+    sleep 1
+done
```

Comment:
Is there a reason for this hot loop?

Reply:
@AardhynLavender. There was a reason. I could not find the way to stop spider printing scraped data into terminal. It supposed to give you some time to check if the data for the first spider was(not) sent to `db` before the message gets pushed up from the logging of the second spider. As I said, I have found the way to stop logging scraped data into terminal, and I will get rid of this loop.